src/commands/instances/moderation/DenyNameChange.ts

```diff
@@ -1,36 +1,30 @@
-import { MessageEmbed } from 'discord.js';
+import { CommandInteraction, GuildMember, MessageEmbed } from 'discord.js';
+import { SlashCommandBuilder } from '@discordjs/builders';
 import { deny } from '../../../api/modules/names';
 import config from '../../../config';
-import { Command, ParsedMessage } from '../../../types';
+import { Command } from '../../../types';
 import { getEmoji, hasModeratorRole } from '../../../utils/discord';
 import CommandError from '../../CommandError';
 
 class DenyNameChange implements Command {
-  name: string;
-  template: string;
+  slashCommand: SlashCommandBuilder;
 
   constructor() {
-    this.name = 'Deny a name change request';
-    this.template = '!deny-name {nameChangeId}';
+    this.slashCommand = new SlashCommandBuilder()
```

Comment:
While it would be a different task, I'd recommend making a single `NameChange` command:
A user / bot uses `/namechange [old] [new] | [namechangeid]` with buttons below for `approve`/`deny`
This way you don't have to manually look up the id, and a button is all you need.
Guess that would be a new feature though

Reply:
Good suggestion. Would certainly make the process of reviewing and taking action easier than what it is right now.